clang/runtime/dpct-rt/include/math.hpp

```diff
@@ -36,6 +37,43 @@ class vectorized_binary<
 
 template <typename T> bool isnan(const T a) { return sycl::isnan(a); }
 // TODO: Need add more specialization such as bfloat16 version.
+
+template <typename T>
+inline int64_t zero_or_signed_extent(const T val, unsigned bit) {
```

Comment:
Let's remove const in function parameter, which passed by value.

Reply:
fixed